packages/message-input/src/vaadin-message-input-mixin.js

```diff
@@ -92,8 +92,6 @@ export const MessageInputMixin = (superClass) =>
 
       this._buttonController = new SlotController(this, 'button', 'vaadin-button', {
         initializer: (btn) => {
-          btn.setAttribute('theme', 'primary');
```

Comment:
There is an existing issue for this https://github.com/vaadin/web-components/issues/5778. Let me create a separate PR though (and also update Lumo accordingly).

Reply:
Created https://github.com/vaadin/web-components/pull/9680